src/TestFramework/PhpUnit/Config/Builder/InitialConfigBuilder.php

```diff
@@ -141,9 +141,9 @@ private function addJUnitLogger(\DOMDocument $dom, \DOMXPath $xPath): void
 
     private function addCodeCoverageLogger(\DOMDocument $dom, \DOMXPath $xPath): void
```

Comment:
Can we remove the $dom paramenter for this mehtod as wel?

Reply:
I'll make a PR for that because this is another review anyway.